### PR TITLE
Replace %in% to `any()`

### PR DESCRIPTION
**Pull request by PavelBal**
Actually just want to check the build.

### CODE REVIEW

File: appveyor-3.6.yml [hunk 1]
```diff
@@ -40,9 +40,9 @@ build_script:
   - Rscript -e "install.packages('https://github.com/Open-Systems-Pharmacology/rClr/releases/download/v0.9.1-R3/rClr_0.9.1.zip', repos = NULL, type = 'binary')"
   - travis-tool.sh install_deps
   - travis-tool.sh r_binary_install curl
-  - Rscript -e "install.packages('https://ci.appveyor.com/api/projects/open-systems-pharmacology-ci/ospsuite-rutils/artifacts/ospsuite.utils.zip', repos = NULL, type = 'binary')"
+  - Rscript -e "download.file('https://ci.appveyor.com/api/projects/open-systems-pharmacology-ci/ospsuite-rutils/artifacts/ospsuite.utils.zip?pr=false', destfile = 'ospsuite.utils.zip', mode='wb'); install.packages('ospsuite.utils.zip', repos = NULL, type = 'binary')"
+  - Rscript -e "download.file('https://ci.appveyor.com/api/projects/open-systems-pharmacology-ci/tlf-library/artifacts/tlf.zip?pr=false', destfile = 'tlf.zip', mode='wb'); install.packages('tlf.zip', repos = NULL, type = 'binary')"  
   - Rscript -e "install.packages(c('ggplot2', 'patchwork', 'vdiffr'), repos = 'http://cran.us.r-project.org')"
-  - Rscript -e "install.packages('https://ci.appveyor.com/api/projects/open-systems-pharmacology-ci/tlf-library/artifacts/tlf.zip', repos = NULL, type = 'binary')"
```
Comment (Yuri05) [inline appveyor-3.6.yml:45]: I think we should extract all R installation instructions (`Rscript -e "..."`) into e.g. **install_OSP_Packages.R** which would be called from both appveyor and appveyor-nightly scripts

Comment (PavelBal) [reply appveyor-3.6.yml:45]: @Felixmil Could you take care of it?

Comment (Felixmil) [reply appveyor-3.6.yml:45]: Sure !

Comment (Felixmil) [reply appveyor-3.6.yml:45]: Happening in #1219

**Comment by PavelBal** [comment]
@Yuri05 @Felixmil Please re-review so it can be merged.

**Review by Felixmil — APPROVED** [review]

**Pull request merged**